### DISCUSSION TITLE
SNES DUMP

### DISCUSSION THREAD

**Discussion by gatarib** [sanni/cartreader#73] [Need help]
I tried to dump Pikiinya! (Japan), but could not. I have no knowledge of programming. I'm sorry. Can someone please help me?

**Comment by sanni** [comment]
Can you upload an image of the PCB? Just drag and drop the image into the message box here.

If it's a standard PCB then cleaning the contacts might be all that is needed. The Cart Reader is very sensitive to oxidation on the gold fingers of the game.

**Comment by gatarib** [comment]
Thanks for the reply, here is the PCB image.
![DSC_1693](https://user-images.githubusercontent.com/77560501/104850547-1d04f380-5933-11eb-9c32-7f1dd5f59447.jpg)
![DSC_1692](https://user-images.githubusercontent.com/77560501/104850553-21c9a780-5933-11eb-9509-d780ea1dcf84.jpg)

**Comment by sanni** [comment]
Ok this is a pretty standard lorom PCB so dumping should work flawlessly. I flashed the ROM to a repro and confirmed that is does dump without errors.
I don't see any bad traces on your PCB so I would wipe the contacts with Isopropanol/rubbing alcohol to make sure they are clean and then insert it a few times into the Cart Reader to clean the contacts of the Cart Reader too.

Also make sure you have somewhat recent version of the Cart_Reader.ino flashed since the game has an all Japanese name which results in an error with firmware older than 2018.

**Comment by gatarib** [comment]
The version of the software I am using is 5.5. I've tried several times, but I get this screen. There seems to be nothing more I can do, so I'm giving up. Thanks for the advice.
![pikinya PCB3](https://user-images.githubusercontent.com/77560501/104922121-9c4f0180-59dd-11eb-8ca9-7c0a136fff6a.jpg)
